### ISSUE TITLE
Wavenumber notation

### ISSUE THREAD

**Issue by navidcy** [FourierFlows/FourierFlows.jl#236]
Currently `grid` contains arrays for (`k`, `l`). Should we change to (`kx`, `ky`)? I'd vote "yes".

```
grid.kx
grid.kxr
grid.ky
grid.ksq
grid.ksqr
grid.invksq
grid.invksqr
```

One would be tempted to use unicode, e.g.

```julia
grid.k²
grid.k²r
grid.k⁻²
grid.k⁻²r
```

but that makes it rather cumbersome for new users...

Any thoughts?

@glwagner,  @BrodiePearson, @liasiegelman

(The docs at the moment have a mixture of the two so it would be good to smooth it out after we settle on the issue.)

**Comment by liasiegelman** [comment]
@navidcy I agree that kx and ky is better than k and l. I prefer without unicode though.
